auto_nag/scripts/topcrash_bad_severity.py

```diff
@@ -57,11 +57,13 @@ def get_bz_params(self, date):
                 "enhancement",
             ],
             "keywords": "topcrash",
-            "keywords_type": "allwords",
```

Comment:
What is the default `keywords_type`?

Reply:
The default is `equals`,  which will not do partial matching (e.g., matching `topcrash-thunderbird` or `topcrash-win`)

Reply:
Won't this exclude "topcrash-startup"?

Reply:
Yes, this will exclude `topcrash-startup`. We could explicitly add `topcrash-startup`, but `topcrash` and `topcrash-startup` should always be together.

Reply:
[Bug 1571516](https://bugzilla.mozilla.org/show_bug.cgi?id=1571516) is the only case where we have the `topcrash-startup` keyword, but we do not have the `topcrash` keyword.

It will not hurt to explicitly match the `topcrash-startup` keyword.

Reply:
Done in c69655002a82a8dd38ff0a181ae7b45a9a6079df